auslib/test/admin/views/test_releases.py

```diff
@@ -689,3 +701,38 @@ def testPostRevisionRollbackBadRequests(self):
 
         ret = self._post(url)
         self.assertEquals(ret.status_code, 400)
+
+    def testSettings(self):
+        # let's poke Balrog about a new settings we got
+        blob = {
+            "name": "settings",
+            "schema_version": 2000,
+            "settings": {
+                "preloaded-sts-pkp": {
+                    "version": "1",
+                    "last_modified": 1438097376
+                }
+            }
+        }
+
+        data = {'name': 'settings', 'version': '1',
+                'product': 'settings', 'blob': json.dumps(blob)}
+
+        ret = self._put('/releases/settings', data=data)
+        self.assertEquals(ret.status_code, 201)
+
+        select_rel = dbo.releases.t.select()
+        r = select_rel.where(dbo.releases.name=='settings')
+        r = r.execute().fetchall()
+        self.assertEquals(len(r), 1)
+        rec = r[0]
+        self.assertEquals(rec['name'], 'settings')
+        self.assertEquals(rec['version'], '1')
+        self.assertEquals(rec['product'], 'settings')
+        self.assertDictEqual(byteify(json.loads(rec['data'])),
```

Comment:
> A normal assertEquals should be fine here.

Sure

> What's the purpose of the byteify'ing?

To compare bytes with bytes, since the json encoding/decoding returns unicode when you initially have strings. e.g.

```python
>>> a = {'aé': '1'}; b = {u'aé': u'1'}
>>> a == b
False
```